apps/web/src/pages/integrations/components/IntegrationSecretTextarea.tsx

```diff
@@ -14,7 +14,7 @@ export const IntegrationSecretTextarea = ({
   credential: IConfigCredentials;
   errors: any;
   field: any;
-  register: any;
+  register?: any;
```

Comment:
Here can't we use the provider type?

Reply:
wdym?

Reply:
As far as I understand from the code in `register` we pass all the user defined information that defines the provider, mainly the credentials of the integration, that then is saved or updated. That should be typed.